<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="29466" started-at="2020-05-23T22:51:03Z" finished-at="2020-05-23T22:51:33Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29466" started-at="2020-05-23T22:51:03Z" finished-at="2020-05-23T22:51:33Z">
      <class name="com.crm.qa.testcases.HomePageTest">
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.crm.qa.testcases.HomePageTest@29f69090]" name="Setup" is-config="true" duration-ms="13607" started-at="2020-05-23T22:51:03Z" finished-at="2020-05-23T22:51:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="VerifyHompageTitleTest()[pri:1, instance:com.crm.qa.testcases.HomePageTest@29f69090]" name="VerifyHompageTitleTest" duration-ms="12" started-at="2020-05-23T22:51:17Z" finished-at="2020-05-23T22:51:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyHompageTitleTest -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.crm.qa.testcases.HomePageTest@29f69090]" name="teardown" is-config="true" duration-ms="774" started-at="2020-05-23T22:51:17Z" finished-at="2020-05-23T22:51:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.crm.qa.testcases.HomePageTest@29f69090]" name="Setup" is-config="true" duration-ms="12870" started-at="2020-05-23T22:51:18Z" finished-at="2020-05-23T22:51:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="VerifyUserNameTest()[pri:2, instance:com.crm.qa.testcases.HomePageTest@29f69090]" name="VerifyUserNameTest" duration-ms="1448" started-at="2020-05-23T22:51:30Z" finished-at="2020-05-23T22:51:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Not Matched] but found [Pranesh Sinha]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Not Matched] but found [Pranesh Sinha]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.crm.qa.testcases.HomePageTest.VerifyUserNameTest(HomePageTest.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyUserNameTest -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.crm.qa.testcases.HomePageTest@29f69090]" name="teardown" is-config="true" duration-ms="728" started-at="2020-05-23T22:51:32Z" finished-at="2020-05-23T22:51:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.crm.qa.testcases.HomePageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
